CONTRIBUTING.md

```diff
@@ -155,14 +155,14 @@ currently using a minimum version of PyCharm 2019.2.4.
     `python -m eland.tests.setup_tests` to setup test environment -*note
     this modifies Elasticsearch indices*
 -   Install local `eland` module (required to execute notebook tests)
-    `python install setup.py`
+    `python setup.py install`
 -   Run `pytest --nbval  --doctest-modules` to validate install
```

Comment:
Future work to do is removing this step, `--doctest-modules` is done within `nox -s test`, just need the `--nbval` as well. Maybe it can be done now in this PR?

Reply:
Off course

Reply:
@mesejo Btw I pushed a nit on the wording to your branch, you may need to do a pull to get my change.